ckanext/unaids/plugin.py

```diff
@@ -253,6 +255,15 @@ def _giftless_upload(context, resource, current=None):
             })
 
 
+def _update_resource_last_modified_date(resource, current={}):
```

Comment:
Default args values should be immutable. https://docs.python-guide.org/writing/gotchas/
Better:
```
def _update_resource_last_modified_date(resource, current=None):
  if not current:
    current = {}
[...]
```